src/app.rs

```diff
@@ -993,6 +1007,12 @@ pub struct Opt {
     #[structopt(short = "s", long = "axis-scale", default_value = "1.0")]
     #[serde(default)]
     pub axis_scale: f32,
+
+    #[structopt(short = "b", long = "move-base-diff-unit", default_value = "0.1")]
+    pub base_translation_diff: f32,
+
+    #[structopt(short = "j", long = "rot-joint-diff-unit", default_value = "0.1")]
```

Comment:
Do you mean `rot` as a rotation? Because joint can be linear, so I think rotation is not suitable.

Reply:
It does!